sys/net/gnrc/link_layer/netdev2/gnrc_netdev2_eth.c

```diff
@@ -0,0 +1,195 @@
+#include "net/gnrc.h"
+#include "net/gnrc/gnrc_netdev2.h"
+#include "net/ethernet/hdr.h"
+
+#include "od.h"
+
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+
+static gnrc_pktsnip_t *_recv(gnrc_netdev2_t *gnrc_netdev2)
```

Comment:
@authmillenon Hm, I don't quite get it.

I think you're mixing up hdr and eth_hdr in the last lines, right?
Where do you mark the payload according to ethertype?